### DISCUSSION TITLE
Bug report: (un)follow buttons not syncronized

### DISCUSSION THREAD

**Discussion by stefan-wullems** [community/community#22060] [New to GitHub]
<p>When I go to someone’s personal profile, there is a big follow/unfollow button on the left:</p>
<p>When I scroll down the page, another follow/unfollow button shows up in a header bar underneath the account picture.</p>
<p>When I follow/unfollow using one of the buttons, the other does not get updated.<br>
<img src="https://user-images.githubusercontent.com/28148115/181068740-ffd35361-4652-40e7-a79a-df2481e64da0.png" alt="image" data-base62-sha1="s2Qg0mKXaMcLhuffCfhnyuXrNYa" width="525" height="215"></p>

**Comment by yamiacat** [comment, marked as answer]
<p>Hi <a class="mention" href="/stefan-wullems">@stefan-wullems</a>! :wave: Welcome to the Community!</p>
<p>Thanks for reporting this! I’ve created an internal issue with the relevant team!</p>
